packages/console/src/pages/SignInExperience/tabs/SignUpAndSignInTab/components/SocialConnectorEditBox/index.tsx

```diff
@@ -0,0 +1,94 @@
+import { ConnectorType } from '@logto/schemas';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+
+import DragDropProvider from '@/components/Transfer/DragDropProvider';
+import DraggableItem from '@/components/Transfer/DraggableItem';
+import useConnectorGroups from '@/hooks/use-connector-groups';
+import type { ConnectorGroup } from '@/types/connector';
+
+import ConnectorSetupWarning from '../ConnectorSetupWarning';
+import AddSocialConnectorButton from './AddSocialConnectorButton';
```

Comment:
What about just "AddButton", since we are in the scope of "SocialSignInForm"

Reply:
Cool, updated.